copyright.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: MIT
+ */
```

Comment:
This file needs a trailing blank line, otherwise it will prepend headers like this:

```javascript
**
 * © 2019 Liferay, Inc. <https://liferay.com>
 *
 * SPDX-License-Identifier: MIT
 */
/**
 * My fancy function
 */
export default function fancy() {}
```

instead of:

```javascript
**
 * © 2019 Liferay, Inc. <https://liferay.com>
 *
 * SPDX-License-Identifier: MIT
 */

/**
 * My fancy function
 */
export default function fancy() {}
```

You need to `.prettierignore` this file to stop Prettier from stripping the trailing blank line though.

Reply:
Makes sense, I'm adding a `.prettierignore` file!